client/src/app.jsx

```diff
@@ -1,26 +1,71 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import ProductCarousel from './components/productCarousel.js';
+import Summary from './components/summary.js';
+import Purchase from './components/purchase.js';
+import Navigation from './components/navigation.js';
+import sample from './sample/sampleProduct.js';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-
+      product: sample
     };
 
     this.handleClick = this.handleClick.bind(this);
   }
 
   handleClick(target) {
-    target = 2;
+    console.log(target);
   }
 
   render() {
     return (
-      <div>React Running</div>
+      <div id="main">
```

Comment:
this DOM tree is pretty verbose. It looks like a couple of the outer div's could be left out or I might not be noticing their utility.

Reply:
Yeah it is pretty verbose. I reviewed the dom tree, and I think I could defend the different element containers I decided to include, although the outermost container divs are the most debatable. Essentially my reasoning was that I wanted to see what the whole proxy server would look like, and then inside that, leave room for any additional headers or footers we might want to add. If I include that structure but don't add a header and footer, then it does sort of make some of the outer divs look unnecessary. If we do add something later though, the structure is there.